includes/Checker/Abstract_Check_Runner.php

```diff
@@ -0,0 +1,179 @@
+<?php
+/**
+ * Class WordPress\Plugin_Check\Checker\Abstract_Check_runner
+ *
+ * @package plugin-check
+ */
+
+namespace WordPress\Plugin_Check\Checker;
+
+use WordPress\Plugin_Check\Checker\Check_Runner;
+use WordPress\Plugin_Check\Checker\Check_Result;
+use WordPress\Plugin_Check\Checker\Preparations\Universal_Runtime_Preparation;
```

Comment:
Nit-pick, you can simplify this as a relative import I think.
```suggestion
use Preparations\Universal_Runtime_Preparation;
```

Reply:
@felixarntz after updating this I had errors when running tests saying the class could not be found? Not entirely sure why?